src/Sylius/Bundle/AdminBundle/Resources/config/services/twig_component.xml

```diff
@@ -46,6 +46,7 @@
 
         <service id="Sylius\Bundle\AdminBundle\TwigComponent\Country\CountryType">
             <argument type="service" id="form.factory" />
+            <argument>Sylius\Bundle\AddressingBundle\Form\Type\CountryType</argument>
```

Comment:
Can we use the resource's form class parameter here instead of hardcoding fqcn?

Reply:
We don't have resource's form class in the container. I asked @loic425 about it, and he created [this](https://github.com/Sylius/SyliusResourceBundle/pull/846) PR.

Reply:
Just checked and yup, the `form` node in AddressBundle's resource configuration is confusing